### PR TITLE
Test if pv-pool backingstore reaching rejected phase in FIPS cluster

### PR DESCRIPTION
**Pull request by mashetty330**

### CODE REVIEW

File: tests/functional/object/mcg/test_pv_pool.py [hunk 1]
```diff
@@ -344,3 +348,37 @@ def test_ephemeral_for_pv_bs(self, backingstore_factory):
         assert (
             "test_object" in search_output
         ), "Dummy data was not found in the node ephemeral storage!"
+
+    @fips_required
+    @tier2
+    @bugzilla("2247731")
+    @polarion_id()
```
Comment (keesturam) [inline tests/functional/object/mcg/test_pv_pool.py:355]: No polarion id?

File: tests/functional/object/mcg/test_pv_pool.py [hunk 2]
```diff
@@ -344,3 +348,37 @@ def test_ephemeral_for_pv_bs(self, backingstore_factory):
...
+            "OC",
+            {"pv": [(1, MIN_PV_BACKINGSTORE_SIZE_IN_GB, CEPHBLOCKPOOL_SC)]},
+        )[0]
+
+        # restart noobaa-core pod
+        get_noobaa_core_pod().delete()
+
+        # wait for about 10 mins to check if
+        # the backingstore has reached Rejected state
+        time.sleep(10)
```
Comment (keesturam) [inline tests/functional/object/mcg/test_pv_pool.py:373]: shouldn't this be 600 if it is 10 minutes?
Also, can you pls add a log.info saying the test is waiting for backingstore to be ready.

File: tests/functional/object/mcg/test_pv_pool.py (hunk 1 above)
Comment (mashetty330) [reply tests/functional/object/mcg/test_pv_pool.py]: done

File: tests/functional/object/mcg/test_pv_pool.py (hunk 2 above)
Comment (mashetty330) [reply tests/functional/object/mcg/test_pv_pool.py]: 10 minutes wait is done in the code below this, where we wait for backingstore state. This 10 seconds wait was given for testing purpose.

**Review by sagihirshfeld — APPROVED** [review]
LGTM

File: tests/functional/object/mcg/test_pv_pool.py [hunk 3]
```diff
@@ -346,3 +349,36 @@ def test_ephemeral_for_pv_bs(self, backingstore_factory):
...
+        # wait for about 10 mins to check if
+        # the backingstore has reached Rejected state
+        pv_bs_obj = OCP(
+            kind=constants.BACKINGSTORE,
+            namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            resource_name=pv_backingstore.name,
+        )
+        assert pv_bs_obj.wait_for_resource(
+            condition="Ready", column="PHASE", timeout=600, sleep=5
+        ), "Pv pool backingstore reached rejected phase after noobaa core pod restart"
```
Comment (keesturam) [inline tests/functional/object/mcg/test_pv_pool.py:381]: @mashetty330 if this test fails, wouldn't it put the test cluster into a bad shape?

Comment (mashetty330) [reply tests/functional/object/mcg/test_pv_pool.py:381]: No it shouldn't, if it does then there is bug 😄

**Review by keesturam — APPROVED** [review]

**Pull request merged**